### ISSUE TITLE
Thumbnail and Preview Images not found.

### ISSUE THREAD

**Issue by Yavieh** [imagegenius/docker-immich#447]
Hey, im currently facing a problem. From the date 28 september, my thumbnails and preview images are not loading any more.
![image](https://github.com/user-attachments/assets/1854e5ef-84a1-4fa0-b43b-6d60cc981fed)

On my device where the pictures where uploaded, the thumbnails and preview image are showing.
Also when i download the file in the browser its working fine. 

This message i get in the browser console when i want to look at a preview  `GET https://mydomain.com/api/assets/5188be6b-c9bc-4363-93c5-320956cb18dd/thumbnail?size=preview&c=O0iewl2RbjAZTVJbRP6uDINtxAQ%3D 404 (Not Found)`

Im updating my immich docker container always to the newest version, so im not sure with what version exactly it startet as im not always looking into the browser version.

Is there any more info i can share like logs? 
Thanks for help.

**Comment by Yavieh** [comment]
I was able to fix the problem with the help of the immich discord. ([link to post](https://discord.com/channels/979116623879368755/1292554830711361616))
TLDR: The issue was, my immich server was running on port 80 and not 8080.
